lang/po/ar.po

```diff
@@ -155548,7 +155548,7 @@ msgstr[5] ""
 #: lang/json/gun_from_json.py
 msgid ""
 "Emphasizing portability and agility over power, this all wood bow is "
-"suitable for small game and harrasing enemies."
```

Comment:
`lang/po/xx.po` files are not supposed to be edited by hand.

Reply:
Oh. Welp. Let me kill these and go again. Sorry